static/modules/collision.js

```diff
@@ -115,39 +114,155 @@ export function isColliding(shapeA, shapeB) {
  * @returns {Vector}
  */
 export function collide(shapeA, shapeB, resolve = true) {
-  if (shapeA instanceof Box && shapeB instanceof Box) {
+  if (shapeA instanceof CollisionBox && shapeB instanceof CollisionBox) {
```

Comment:
To be honest, I'm not a huge fan of this rename since it wasn't really ambiguous for us before, and it makes the collision code kind of more annoying to read. I wonder what @Matt-Puentes thinks.

Reply:
It was confusing to me since `circle` is a function used to draw circles on the canvas and `Circle` is a class used for calculating collisions.

Reply:
The `circle` and `Circle` naming collision is unfortunate and @jojonium 's solution is probably the best one